app/src/main/java/org/oppia/android/app/resumelesson/ResumeLessonActivity.kt

```diff
@@ -0,0 +1,104 @@
+package org.oppia.android.app.resumelesson
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+import org.oppia.android.app.home.RouteToExplorationListener
+import org.oppia.android.app.player.exploration.ExplorationActivity
+import javax.inject.Inject
+
+class ResumeLessonActivity :
```

Comment:
Here & everywhere else in the PR: please add KDocs to all public classes, vals, and methods (for vals that are non-inject properties of classes, please use ``@property`` clauses in the class's KDoc).